src/LessMsi.Cli/Program.cs

```diff
@@ -56,14 +58,18 @@ public static int Main(string[] args)
 				 * See https://github.com/mono/mono/blob/master/mcs/tools/mdoc/Mono.Documentation/mdoc.cs#L54  for an example of using "commands" and "subcommands" with the NDesk.Options lib.
 				 */
 
+                ExtractCommand extractCommandObject = new ExtractCommand();

```

Comment:
remove the `...Object` postfix on the variable name pls.

Reply:
Will amend the code accordingly.